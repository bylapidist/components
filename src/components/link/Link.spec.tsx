import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Link } from './index';

test('it works', () => {
    const tree = renderer.create(<Link href="#" title="venenatis" />).toJSON();
    expect(tree).toMatchSnapshot();
});
