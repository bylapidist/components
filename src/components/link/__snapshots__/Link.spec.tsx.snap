// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  border-style: solid;
  font-size: inherit;
  font-weight: 400;
  font-family: Montserrat,sans-serif;
  line-height: 2;
  -webkit-letter-spacing: 0.025em;
  -moz-letter-spacing: 0.025em;
  -ms-letter-spacing: 0.025em;
  letter-spacing: 0.025em;
  border-bottom-width: 2px;
  color: #1a8385;
  border-bottom-color: #4AD6D8;
}

.c0:hover {
  color: #1a8385;
  border-bottom-color: #1a8385;
}

<a
  className="c0"
  href="#"
>
  Hello world
</a>
`;
